pvanalytics/system.py

```diff
@@ -401,3 +403,172 @@ def infer_orientation_daily_peak(power_or_poa, sunny, tilts,
                 best_azimuth = azimuth
                 best_tilt = tilt
     return best_azimuth, best_tilt
+
+
+def _power_residuals_from_clearsky(system_params,
+                                   ghi, dhi, dni,
+                                   solar_zenith, solar_azimuth,
+                                   power_ac, temperature,
+                                   wind_speed,
+                                   temperature_coefficient,
+                                   temperature_model_parameters):
+    # Return the residuals between a system with parameters given in `params`
```

Comment:
why comment the doc string?

Reply:
I've generally followed the convention that only functions in the public API of a module have docstrings, private functions get block comments. I think PEP-8 hints at this, but doesn't demand one way or another. 

I think I mainly do it because I sometimes use another language where you get a compiler warning if you attach documentation to a non-public function. Do you have a preference?

Reply:
I prefer regular docstrings so that my IDE renders it correctly. 

PEP 8 says "Docstrings are not necessary for non-public methods, but you should have a comment that describes what the method does." I interpret that as "if you're not going to write a doc string for your private function, at least make a brief comment."

Reply:
That's a reasonable interpretation. I actually prefer comments to docstrings for the same reason. My docstring highlighting color is brighter and I like not having private functions jump out at me when I'm reading through unfamiliar code. I'm fine with changing it, since I suspect I'm in the minority here.